src-a1/main/java/com/example/a1/controller/rest/VoteController.java

```diff
@@ -0,0 +1,29 @@
+package com.example.a1.controller.rest;
+
+import com.example.a1.service.AnswerManagementService;
+import com.example.a1.service.QuestionManagementService;
+import com.example.a1.service.UserManagementService;
+import com.example.a1.service.VoteManagementService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequiredArgsConstructor
+public class VoteController {
+    private final VoteManagementService voteManagementService;
+    private final UserManagementService userManagementService;
+
+    @PostMapping("/questions/{questionId}/answers/{answerId}/votes/{voteValue}")
```

Comment:
It would be better to pass the "vote value" as part of a small DTO inside the body inside the request body and not the path parameter.